conf/mesos-site.xml

```diff
@@ -38,18 +48,13 @@
   </property>
 
   <property>
-    <name>mesos.hdfs.executor.uri</name>
-    <value>http://path/to/executor</value>
-  </property>
-
-  <property>
     <name>mesos.hdfs.cluster.name</name>
     <value>hdfs-super-available</value>
```

Comment:
I think that's a clever name and we will have it working pretty soon. :) Did you have some alternate suggestions?